shared/locales/media/media.en-US.properties

```diff
@@ -0,0 +1,3 @@
+playbackPlay.ariaLabel=Play
```

Comment:
This is the standard way to use it. You could localize other attributes like title and alt in the same way.